server/app/tests/data_harvester_test.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+import os
+from typing import List
+from src.data_harvester import SubjectSnapshot
+
+class DataHarvesterTest(unittest.TestCase):
+
+    def test_discard_short_trials(self):
+
+        # Dir for all short trials
+        datadir = '/Users/janellekaneda/Documents/AddBiomechanics/server/app/test_data/data_harvester_test_short'
```

Comment:
This needs to become a relative path, otherwise it won't work on other people's computers. Eventually when we set up GitHub to run all our tests every time we push any code, we will also want it to work there.

So we can do something like `datadir = os.path.abspath('../test_data/data_harvester_test_short')`.

Also, we need to include the necessary files in the GitHub so other people can run these tests when they clone the repo, so those need to be `git add`ed and pushed.